docs/panels.md

```diff
@@ -30,3 +30,12 @@ Panels can be reordered simply by dragging them into the desired position.
 3. __Save__ the new name by pressing <kbd>Enter</kbd> or clicking ![The tick icon](images/panels/savePanelRename.png?raw=true) at the top right corner of the panel.
 
 To __abort__ the renaming action any time before step 3, press <kbd>Esc</kbd> or click ![The undo arrow icon](images/panels/cancelPanelRename.png?raw=true) at the top right corner of the panel.
+
+## Watch List Panels <a name="watchlist"></a>
+
+Watch list panels are panels whose filter reflects a "watch list". More specifically, a filter reflects a watch list if either of the following conditions is met :
+
+- It is empty (in other words, it watches all issues from the primary repo)
+- It is in the form of `id:repo1#id1;repo2#id2;repo3#id3;...;repoN#idN` (you can refer [here](filters.md) for more explanation about this filter syntax)
```

Comment:
I just realised that I haven't added the relevant documentation for shorter form of OR to the documentation :sweat_smile:

Can I do it here? Or should I do it as separate issue?